packages/backend-app-api/src/config/config.ts

```diff
@@ -70,6 +71,7 @@ export async function createConfigSecretEnumerator(options: {
 export async function loadBackendConfig(options: {
   remote?: LoadConfigOptionsRemote;
   argv: string[];
+  additionalConfig?: AppConfig;
```

Comment:
I'm feeling we might as well make this `AppConfig[]`, so that you can provide context for config loaded from multiple different external sources.

Reply:
Good idea.